WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewModel.swift

```diff
@@ -37,9 +37,7 @@ final class InPersonPaymentsMenuViewModel {
         cardPresentPaymentsConfiguration.isSupportedCountry
     }
 
-    var isEligibleForTapToPayOnIPhone: Bool {
-        return isEligibleForCardPresentPayments && cardPresentPaymentsConfiguration.supportedReaders.contains { $0 == .appleBuiltIn }
-    }
+    @Published var isEligibleForTapToPayOnIPhone: Bool = false
```

Comment:
I think we can restrict access to `isEligibleForTapToPayOnIPhone`, right?
```suggestion
    @Published private(set) var isEligibleForTapToPayOnIPhone: Bool = false
```

Reply:
Yep, thanks, I never remember that. I'll do it in a follow on PR